docs/source/notebooks/iv_pymc.ipynb

```diff
@@ -0,0 +1,4154 @@
+{
```

Comment:
<p>Feels like we need some kind of summary</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/CausalPy/pull/213/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<img width="746" alt="image" src="https://github.com/pymc-labs/CausalPy/assets/10588167/ddfb9f26-16e7-4a15-8648-0f644a6059c4">